diff --git a/android/CMakeLists.txt b/android/CMakeLists.txt
index d4731e6c9e5b7cfee8c14dcd7c34639e4d3842d6..1cfe7f6d85c6a01dda01d120a2d4dbaad4d7b9c1 100644
--- a/android/CMakeLists.txt
+++ b/android/CMakeLists.txt
@@ -11,6 +11,10 @@ set(CMAKE_CXX_STANDARD 20)
 include("${REACT_NATIVE_DIR}/ReactAndroid/cmake-utils/folly-flags.cmake")
 add_compile_options(${folly_FLAGS})
 
+add_link_options(
+  -Wl,--build-id=none
+)
+
 string(
   APPEND
   CMAKE_CXX_FLAGS
