diff --git a/android/build.gradle b/android/build.gradle
index ac149a0e7df5e6ee2a6f839d5fed84c5b24a2175..87bcc81a4e94b03c57cc80231b68dd6e0e8b7b54 100644
--- a/android/build.gradle
+++ b/android/build.gradle
@@ -158,9 +158,13 @@ android {
                     arguments "-DREACT_NATIVE_DIR=${REACT_NATIVE_DIR}",
                             "-DREACT_NATIVE_MINOR_VERSION=${REACT_NATIVE_MINOR_VERSION}",
                             "-DANDROID_STL=c++_shared",
-                            "-DANDROID_SUPPORT_FLEXIBLE_PAGE_SIZES=ON"
+                            "-DANDROID_SUPPORT_FLEXIBLE_PAGE_SIZES=ON",
+                            "-DCMAKE_SHARED_LINKER_FLAGS=-Wl,--build-id=none"
                     abiFilters(*reactNativeArchitectures())
                 }
+                ndkBuild {
+                    arguments "APP_LDFLAGS+=-Wl,--build-id=none"
+                }
             }
         }
     }
