diff --git a/android/CMakeLists.txt b/android/CMakeLists.txt
index 741c501a8548985ed7f71a1326ccf620a0efef5e..54ae40b070d86ccabbf498acd3731d7edbcf4901 100644
--- a/android/CMakeLists.txt
+++ b/android/CMakeLists.txt
@@ -11,6 +11,10 @@ set(CMAKE_CXX_STANDARD 20)
 include("${REACT_NATIVE_DIR}/ReactAndroid/cmake-utils/folly-flags.cmake")
 add_compile_options(${folly_FLAGS})
 
+add_link_options(
+  -Wl,--build-id=none
+)
+
 string(APPEND CMAKE_CXX_FLAGS
        " -DREACT_NATIVE_MINOR_VERSION=${REACT_NATIVE_MINOR_VERSION}\
    -DREANIMATED_VERSION=${REANIMATED_VERSION}")
