diff --git a/node_modules/react-native-emoji-selector/index.js b/node_modules/react-native-emoji-selector/index.js
index 0737acd..a1d1f2c 100644
--- a/node_modules/react-native-emoji-selector/index.js
+++ b/node_modules/react-native-emoji-selector/index.js
@@ -70,6 +70,7 @@ const categoryKeys = Object.keys(Categories);
 
 const TabBar = ({ theme, activeCategory, onPress, width }) => {
   const tabSize = width / categoryKeys.length;
+  if (tabSize - 12 < 1) return;
 
   return categoryKeys.map(c => {
     const category = Categories[c];
@@ -80,7 +81,7 @@ const TabBar = ({ theme, activeCategory, onPress, width }) => {
           onPress={() => onPress(category)}
           style={{
             flex: 1,
-            height: tabSize,
+            height: tabSize + 12,
             borderColor: category === activeCategory ? theme : "#EEEEEE",
             borderBottomWidth: 2,
             alignItems: "center",
@@ -91,7 +92,7 @@ const TabBar = ({ theme, activeCategory, onPress, width }) => {
             style={{
               textAlign: "center",
               paddingBottom: 8,
-              fontSize: tabSize - 24
+              fontSize: tabSize - 12
             }}
           >
             {category.symbol}
